FirebaseCore/Sources/Public/FIREmulatorSettings.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// A string constant representing a Firebase service with an emulator available.
+typedef NSString *FIREmulatorService NS_TYPED_EXTENSIBLE_ENUM NS_SWIFT_NAME(EmulatorSettings);
+
+/** :nodoc: */
+NSString *const FIREmulatorServiceDatabase;
```

Comment:
In Android we ended up putting these in the service SDKs rather than in the common one.  So instead of:

```java
enum FirebaseEmulators {
  DATABASE
  FIRESTORE
}
```

We did:
```java
FirebaseFirestore.EMULATOR
FirebaseDatabase.EMULATOR
```

The reason we did this was because we didn't want to have to update the common SDK every time a new emulator was added and then release all the other SDKs to depend on common again.

Not sure if that's a concern on iOS as well, just throwing it out there.